### PR TITLE
Fortigate Issue

### PR DESCRIPTION
**Pull request by orlichter1**
## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/27992

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Demisto
- [ ] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/FortiGate/Integrations/FortiGate/FortiGate.py [hunk 1]
```diff
@@ -43,7 +43,10 @@ def login():
         'secretkey': PASSWORD,
         'ajax': 1
     }
+    demisto.info("################")
+    demisto.info("Logging into FortiGate")
     session.post(SERVER + url_suffix, data=params, verify=USE_SSL)  # type: ignore
+    demisto.info("Logged into Fortigate")
```
Comment (yaakovi) [inline Packs/FortiGate/Integrations/FortiGate/FortiGate.py:49]: once the issue is resolved, remove those logs (or organize them)

File: Packs/FortiGate/Integrations/FortiGate/FortiGate.py [hunk 2]
```diff
@@ -69,9 +73,14 @@ def http_request(method, url_suffix, params={}, data=None):
         return_error('Error in API call to FortiGate [%d] - %s' % (res.status_code, res.reason))
     if method.upper() != 'GET':
         return res.status_code
+    demisto.info("################################")
+    demisto.info("status_code: "+str(res.status_code))
+    demisto.info("################################")
+    demisto.info("full res: " + res.content)
```
Comment (yaakovi) [inline Packs/FortiGate/Integrations/FortiGate/FortiGate.py:79]: same here

**Review by yaakovi — APPROVED** [review]
once the issue is resolved:
* add release notes
* keep the `logger` usage
* remove extra logs

**Comment by kirbles19** [comment]
@orlichter1 Done

**Pull request merged**